frontend/packages/core/src/popper.tsx

```diff
@@ -89,7 +89,7 @@ export interface PopperProps
   extends Pick<ClickAwayListenerProps, "onClickAway">,
     Pick<MuiPopperProps, "placement"> {
   open: boolean;
-  anchorRef?: React.MutableRefObject<HTMLElement> | null;
+  anchorRef?: React.MutableRefObject<HTMLElement> | React.MutableRefObject<null> | null;
```

Comment:
i think this should work
```suggestion
  anchorRef?: React.MutableRefObject<HTMLElement | null> | null;
```